core/testdata/modrules/patch/patch-13.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: api.kubemod.io/v1beta1
+kind: ModRule
+metadata:
+  name: modrule-1
+  namespace: kubemod-system
+spec:
+  type: Patch
+  targetNamespaceRegex: '\w{2,}-[^ ]{3}[[:lower:]]+?[[:xdigit:]]+'
+  # ^^^ regex to match 'my-namespace' -- the namespace used by the test suite
```

Comment:
Please move comment above the line to make it consistent with the rest of the code.

Reply:
👍